packages/apps/ability-relay-link/src/lib/decodeTransaction.spec.ts

```diff
@@ -0,0 +1,135 @@
+import { DecodeTransactionParams } from '@lit-protocol/vincent-ability-sdk/gatedSigner';
+import { Address, encodeFunctionData } from 'viem';
+
+import { decodeTransaction } from './decodeTransaction';
+import { ERC20_ABI } from './helpers/erc20';
```

Comment:
You can import the ABI from the `ability-sdk`, it's defined at `packages/libs/ability-sdk/src/lib/abilityHelpers/erc20-abi.ts`